js/samples/04.ai-apps/a.teamsChefBot/teamsapp.local.yml

```diff
@@ -50,7 +50,7 @@ deploy:
     # Provides the Teams Toolkit .env file values to the apps runtime so they can be accessed with `process.env`.
     - uses: file/createOrUpdateEnvironmentFile
       with:
-          target: ./.env
+          target: ./.localConfigs
```

Comment:
Prefer to keep using `.env` file. We already have too many places to find variable files. Use either .env file or env folder.